### PR TITLE
plan title should have ellipsis when it's too long to show initially

### PR DESCRIPTION
**Pull request by pandafulmanda**
## Initial
screen shot 2015-06-10 at 5 55 01 pm

## On hover/active
screen shot 2015-06-10 at 5 55 08 pm

### CODE REVIEW

File: src/components/course-calendar/plan.cjsx [hunk 1]
```diff
@@ -85,7 +85,8 @@ CoursePlan = React.createClass
       left: offset * 100 / 7 + '%'
       top: (weekTopOffset + 4 - order * 3) + 'rem'
 
-    planClasses = "plan #{plan.type} course-plan-#{plan.id}"
+    planLabelClass = 'plan-label-long' if plan.title.length > 18
```
Comment (philschatz) [inline src/components/course-calendar/plan.cjsx]: Where does the 18 come from?

**Comment by philschatz** [comment]
Hovering does this to me:
image

**Comment by pandafulmanda** [comment]
Ah, I see.  Hmmm, now it should work correctly.  You may need to make your window skinny and then load the calendar to see the effect.  I decided not to add some detect for window resize, but if you think it's worth it, let me know

File: src/components/course-calendar/plan.cjsx [hunk 2]
```diff
@@ -13,30 +13,44 @@ CoursePlan = React.createClass
...
     samePlans = Array.prototype.slice.call(container.querySelectorAll(classes))
 
-  toggleModal: (mouseEvent, key) ->
+  closePlanOnModalHide: ->
+    hide = @refs.trigger.hide
+    trigger = React.findDOMNode(@refs.trigger)
+    syncClosePlan = @syncClosePlan
+
+    # alias modal hide to also make plan look un-selected
+    @refs.trigger.hide  = ->
```
Comment (philschatz) [inline src/components/course-calendar/plan.cjsx]: Can you use a fat arrow (`=>`) here to avoid the local vars?

**Pull request merged**